Formula/bazelisk.rb

```diff
@@ -0,0 +1,29 @@
+class Bazelisk < Formula
+  desc "User-friendly launcher for Bazel"
+  homepage "https://github.com/bazelbuild/bazelisk/"
+  url "https://github.com/bazelbuild/bazelisk.git",
```

Comment:
I should note that I'm using git instead of an archive as the version stamping script uses git tags to stamp the version.